samples/02_power_users_developers/count_the_usage_frequency_of_api_methods.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>Looks like both Shannon and the learn lesson team like to differentiate our users into <code>GIS Professionals</code> and <code>Data Scientists</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/514/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>